seurat-filter-cells.R

```diff
@@ -96,6 +97,20 @@ suppressPackageStartupMessages(require(Seurat))
 
 filtered_seurat_object <- FilterCells(seurat_object, subset.names = subset_names, low.thresholds = lt, high.thresholds = ht, cells.use = cells_use)
 
+# Print a summary of the affects of filtering
+
+opt_table <- data.frame(value=unlist(opt), stringsAsFactors = FALSE)[c(-1,-5,-6), , drop = FALSE]
```

Comment:
It's a bit difficult to see what `c(-1, -5, -6)` excludes, perhaps using something more explicit or adding comment for better readability?

Reply:
Thanks @nh3 - I've added a commit to address that. Look good?

Reply:
Awesome- can I get approval then?